flang/test/Semantics/OpenMP/interop-construct.f90

```diff
@@ -0,0 +1,25 @@
+!RUN: not %flang_fc1 -fopenmp %s 2>&1 | FileCheck %s
+
```

Comment:
Add which OpenMP version and what you are doing this test.

See other tests for reference in the same folder

Reply:
I have taken reference from llvm-project/flang/test/Semantics/OpenMP/invalid-branch.f90 to create this semantic test file.

Reply:
Use this reference: https://github.com/swatheesh-mcw/llvm-project/blob/main/flang/test/Semantics/OpenMP/allocate-clause01.f90

Reply:
`error: Semantic errors in llvm-project/flang/test/Semantics/OpenMP/interop-construct.f90
error: Each interop-var may be speciﬁed for at most one action-clause of each interop construct.
error: Each interop-type may be speciﬁed at most once.
error: A depend clause can only appear on the directive if the interop-type includes targetsync`